dd-sdk-android-core/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -249,6 +272,11 @@ object Datadog {
      */
     @JvmStatic
     @JvmOverloads
+    @Deprecated(
+        "UserInfo id property is now mandatory.",
+        ReplaceWith("setUserInfo(id!!, name, email, extraInfo, sdkCore)")
```

Comment:
I think we shouldn't put `ReplaceWith` here, because it uses force-unwrap. Essentially method name is the same, and user needs to adjust just call parameter.